src/index.html

```diff
@@ -7,12 +7,66 @@
       content="width=device-width, initial-scale=1.0"
     />
     <title>Product cards</title>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+
     <link
       rel="stylesheet"
       href="./styles/index.scss"
     />
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div
+      class="card"
+      data-qa="card"
+    >
+      <div class="card__top">
+        <a
+          href="#"
+          class="card__image"
+        >
+          <img
+            src="images/imac.jpeg"
+            alt="photo of an apple monitor"
+            class="card__imac"
+          />
+        </a>
+      </div>
+
+      <div class="card__bottom">
+        <p class="card__title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </p>
+        <p class="card__subtitle">Product code: 195434</p>
+        <div class="card__quality">
+          <div class="stars stars--4">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+          <p class="card__reviews">Reviews: 5</p>
+        </div>
+        <div class="card__cost">
+          <p class="card__price">Price:</p>
+          <p class="card__sum">$2,199</p>
+        </div>
+        <a
+          href="#"
```

Comment:
The 'href' attribute currently points to '#', which is a placeholder. Consider updating it to point to a valid URL or remove it if the link is not needed.